source/MaterialXCore/Element.h

```diff
@@ -579,9 +600,8 @@ class Element : public enable_shared_from_this<Element>
 
     /// Copy all attributes and descendants from the given element to this one.
     /// @param source The element from which content is copied.
-    /// @param sourceUris If true, then source URIs from the given element
-    ///    and its descendants are also copied.  Defaults to false.
-    void copyContentFrom(ConstElementPtr source, bool sourceUris = false);
+    /// @param copyOptions Optional pointer to element copying options object.
```

Comment:
Here's a more detailed description we could use here, following the language used to describe XmlReadOptions arguments:

```
/// @param copyOptions An optional pointer to a CopyOptions object.
///    If provided, then the given options will affect the behavior of the
///    copy function.  Defaults to a null pointer.
```